extensions/markdown-language-features/notebook/index.ts

```diff
@@ -177,41 +177,39 @@ export const activate: ActivationFunction<void> = (ctx) => {
 		hr {
 			border: 0;
 			height: 2px;
-			border-bottom: 2px solid;
 		}
 
-		h2, h3, h4, h5, h6 {
-			font-weight: normal;
+		hr,
+		h1,
+		h2 {
+			border-bottom: 1px solid var(--vscode-notebook-cellBorderColor);
```

Comment:
IMHO we don't want more borders in notebook editor

Reply:
Gotcha, I removed the border.